src/client/Microsoft.Identity.Client.Broker/WamAdapters.cs

```diff
@@ -281,5 +288,30 @@ private static string GetExpectedRedirectUri(string clientId)
         {
             return $"ms-appx-web://microsoft.aad.brokerplugin/{clientId}";
         }
+
+        /// <summary>
+        /// Validate common auth params
+        /// </summary>
+        /// <param name="authenticationRequestParameters"></param>
+        /// <param name="logger"></param>
+        /// <exception cref="MsalClientException"></exception>
+        public static void ValidateAuthParams(
+            AuthenticationRequestParameters authenticationRequestParameters,
+            ILoggerAdapter logger)
+        {
+            string[] oidcScopes = { "offline_access", "openid", "profile" };
+
+            //MSAL Runtime throws an ApiContractViolation Exception with Tag: 0x2039c1cb (InvalidArg)
```

Comment:
- So when does this error happen exactly? The referenced issue implies when SighOut is called? RemoveAccount/SignOut doesn't even accept scopes. For me error happened when calling ATI without scopes.
- I actually get a different error when calling SignOut for lab account (maybe expected?).
```text
msidlabmsa1001@outlook.com login.windows.net 9188040d-6c67-4c5b-b112-36a304b66dad
Exception: MSAL.NetCore.4.46.0.0.MsalServiceException: 
	ErrorCode: wam_failed_to_signout
Microsoft.Identity.Client.MsalServiceException: Failed to sign out account. Status: Unexpected
Context: Failed to get WebAccount from auth parameters
Tag: 0x1f8c55e3
```

Reply:
there are 3 bugs here, first issue is when calling into Runtime Broker ATI / ATS without scopes then we hit the `GlobalExceptionHandler: Status: ApiContractViolation Context: (pii) Tag: 0x2039c1cb`, which will be fixed by this PR. 

Second, The Sign-out issue is already tracked in MSALRuntime and @MSamWils will be fixing that. 

Third, in WPF the sign-in and sign-out flow easily repros the "garbage collection issue", which has been fixed and you can test it with the new Interop NuGet [here](https://identitydivision.visualstudio.com/IDDP/_build/results?buildId=988202&view=artifacts&pathAsName=false&type=publishedArtifacts). 

i will keep this comment open for @MSamWils to look at the Sign-Out issue.

`Microsoft.Identity.Client.MsalServiceException: Failed to sign out account. Status: Unexpected
Context: Failed to get WebAccount from auth parameters
Tag: 0x1f8c55e3`

Reply:
@MSamWils the other issue has been closed file one for [SignOut exception](https://identitydivision.visualstudio.com/Engineering/_workitems/edit/2066797) 

SignOut API result is Success/Failure. But there seems to be a MSARuntime exception thrown

Reply:
Closing this as all issues addressed in the original bug has been fixed